### PR TITLE
Revamp rule details header

### PR DESCRIPTION
**Pull request by alex-shnayder**
screenshot-55

### CODE REVIEW

**Comment by dzimine** [comment]
@alex-shnayder pls take a look https://stackstorm.slack.com/archives/ui-ux/p1442951276000941

**Comment by alex-shnayder** [comment]
@dzimine Can't see it, probably no access.

**Comment by enykeev** [comment]
The minor polish would be to compensate the paddings in proportion 16px 20px 16px (http://cloud.mech.sh/dJEq) rather than current 20px 20px 11px (http://cloud.mech.sh/dK4Z), but the rest seems fine by me.

File: modules/st2-pack-icon/style.less [hunk 1]
```diff
@@ -0,0 +1,44 @@
+@size: 22px;
```
Comment (enykeev) [inline modules/st2-pack-icon/style.less]: Hey, wait a second, who said it is 22? Nope. Despite @gmedrano artistic vision, 32px icons are non negotiable.

**Comment by enykeev** [comment]
diff

**Comment by emedvedev** [comment]
Smaller icons look better, but I side with @enykeev on this one: we can't expect browser scaling to look good every time, so we have to resort to natural size which is 32x32. That said, everything else looks good.

**Pull request merged**